CHANGELOG.rst

```diff
@@ -1,26 +1,89 @@
-Version 0.3.0 (coming soon...)
-------------------------------
-
-.. warning::
-    This release is NOT backwards compatable with v0.2.3. Specifically we have
-    removed the ``get_parent()``, ``get_children()``, and
-    ``get_sentence_generator()`` functions from the classes inhereting from
-    Context in the data model.
-
+[Unreleased]
```

Comment:
Made updates to the CHANGELOG to follow [Keep A Changelog](https://keepachangelog.com/en/1.0.0/) conventions

Reply:
👍